frontend/src/components/eMIB/EditEmail.jsx

```diff
@@ -3,6 +3,9 @@ import PropTypes from "prop-types";
 import LOCALIZE from "../../text_resources";
 import { EMAIL_TYPE, actionShape } from "./constants";
 
+const MAX_RESPONSE = "500";
+const MAX_REASON = "100";
```

Comment:
Can you put more significant names here, since this can be confusing and / or a comment?

Reply:
Comments would be great!

Reply:
Added comments in af57a2c